base/bucket_gocb.go

```diff
@@ -61,8 +61,8 @@ type CouchbaseBucketGoCB struct {
 
 type GoCBLogger struct{}
 
-func (l GoCBLogger) Output(s string) error {
-	LogTo("gocb", s)
+func (l GoCBLogger) Log(level gocbcore.LogLevel, offset int, format string, v ...interface{}) error {
```

Comment:
What do you think about using the gocbcore.LogLevel to switch to log.Warn or log.LogError?